gradle/dependencies.gradle

```diff
@@ -39,8 +39,9 @@ ext {
             appcenter        : '1.4',
             ktlint           : '0.34.0',
             commonsIO        : '2.6',
-            mapboxSdkVersions: '1.0.1',
-            mapboxSdkCore    : '1.8.4'
+            mapboxSdkVersions: '1.1.0',
+            mapboxSdkCore    : '1.8.4',
```

Comment:
Just want to make sure the right version is included 😊

Reply:
Yes, for android gradle plugin v4.0.1, the 1.0.1 does not work with it.
https://app.circleci.com/pipelines/github/mapbox/mapbox-gl-native-android/997/workflows/0634e66a-ca93-483f-bc20-9f2f9a43d764/jobs/2569

Reply:
Yes, this is also included in my integration PR.